templates/configmap.yaml

```diff
@@ -73,8 +73,6 @@ data:
     worker_container_repository = {{ .Values.images.airflow.repository }}
     worker_container_tag = {{ .Values.images.airflow.tag }}
     worker_container_image_pull_policy = {{ .Values.images.airflow.pullPolicy }}
-    worker_service_account_name = {{ .Release.Name }}-worker-serviceaccount
-    image_pull_secrets = {{ template "registry_secret" . }}
```

Comment:
@dimberman why'd we make these changes?

Reply:
@schnie these configs no longer matter when there is a pod_template_file, that said I could keep them in if we plan to keep using this chart for earlier airflow versions? (or maybe we might need to fork?)

Reply:
We could either keep them or use the `semVerCompare` function here too. What do you think?

Reply:
@schnie that works I'll set that up now.